### PR TITLE
NXCM-3801 - Reduce the alarm level of messages while loading nexus.xml

### PR DESCRIPTION
**Pull request by velo**
Is it any better?
https://issues.sonatype.org/browse/NXCM-3801?focusedCommentId=152683&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-152683

Any other suggestions?

### CODE REVIEW

File: nexus/nexus-configuration/src/main/java/org/sonatype/nexus/configuration/source/FileConfigurationSource.java [hunk 1]
```diff
@@ -168,7 +168,7 @@ public Configuration loadConfiguration()
         }
         catch ( ConfigurationException e )
         {
-            getLogger().info( "Configuration file is invalid, attempting upgrade" );
+            getLogger().info( "Configuration file is out to dated, begin upgrade" );
```
Comment (nabcos) [inline nexus/nexus-configuration/src/main/java/org/sonatype/nexus/configuration/source/FileConfigurationSource.java]: IMO this should be debug (and 'outdated').

I would also lower "Upgrading old Nexus configuration file (version 1.4.0)" from DefaultApplicationConfigurationUpgrader to debug level.

Comment (velo) [inline nexus/nexus-configuration/src/main/java/org/sonatype/nexus/configuration/source/FileConfigurationSource.java]: Done

**Comment by nabcos** [comment]
+1

**Comment by cstamas** [comment]
:cake:

**Pull request merged**